src/test/Header.test.js

```diff
@@ -3,7 +3,7 @@ import { render } from '@testing-library/react';
 import { MemoryRouter as Router } from 'react-router-dom';
 import Header from '../components/Header/Header';
 
-it('It should render the component Correctly', () => {
+it('Header component is rendered correctly', () => {
   const header = render(
     <Router>
       <Header />
```

Comment:
- [ ] it would be better if you write tests for redux pure function as required in the requirements in order to meet them all 
- Screenshot of the requirements 
<img width="536" alt="Screenshot 2022-12-10 at 20 33 51" src="https://user-images.githubusercontent.com/61790587/206872366-13e6dd62-05ad-4d56-95ce-95a821a96840.png">